helm-chart-sources/fdk-dataset-harvester/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    fdk.service: fdk-dataset-harvester
+  name: fdk-dataset-harvester
+  namespace: {{ required "Missing value NAMESPACE" .Values.NAMESPACE }}
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        fdk.service: fdk-dataset-harvester
+    spec:
+      containers:
+        - env:
+          - name: RABBIT_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: common-{{ required "Missing value NAMESPACE" .Values.NAMESPACE }}
+                key: RABBITMQ_USERNAME
+          - name: RABBIT_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: common-{{ required "Missing value NAMESPACE" .Values.NAMESPACE }}
+                key: RABBITMQ_PASSWORD
+          - name: SPRING_PROFILES_ACTIVE
+            valueFrom:
+              secretKeyRef:
+                name: common-{{ required "Missing value NAMESPACE" .Values.NAMESPACE }}
+                key: SPRING_PROFILES_ACTIVE
+          - name: emailUsername
```

Comment:
Secret: FDK_DATASET_HARVESTER
keyref=det samme navnet som du bruker i koden

Reply:
dvs "name: FDK_DATASET_HARVESTER-{{ required "Missing value NAMESPACE" .Values.NAMESPACE }}"?

Er litt forskjell fra normal navngiving for secrets

Reply:
fdk-dataset-harvester-[namespace]